# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libpng/pngconf_setjmp.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

At least inkscape suffers from this forced build error.
-- mtr

--- libpng-1.2.39/pngconf.h.orig	2009-08-13 13:13:21.000000000 +0200
+++ libpng-1.2.39/pngconf.h	2009-10-06 07:15:35.000000000 +0200
@@ -335,8 +335,8 @@
        /* If you encounter a compiler error here, see the explanation
         * near the end of INSTALL.
         */
-           __pngconf.h__ in libpng already includes setjmp.h;
-           __dont__ include it again.;
+       /*    __pngconf.h__ in libpng already includes setjmp.h;
+           __dont__ include it again.; */
 #      endif
 #    endif /* __linux__ */
 #  endif /* PNG_SKIP_SETJMP_CHECK */
